desktop_app/src/backend/sandbox/podman/runtime/index.ts

```diff
@@ -153,11 +153,7 @@ export default class PodmanRuntime {
   }
 
   async pullBaseImageOnMachineInstallationSuccess(machineSocketPath: string) {
-    try {
-      await this.baseImage.pullBaseImage(machineSocketPath);
-    } catch (error) {
-      throw error; // Re-throw to be handled by caller
-    }
+    await this.baseImage.ensureBaseImageAvailable(machineSocketPath);
```

Comment:
this should still be wrapped in the `try`/`catch` as it was, I would just update this line:

```diff
-- await this.baseImage.pullBaseImage(machineSocketPath);
++ await this.baseImage.ensureBaseImageAvailable(machineSocketPath);
```